src/node_util.cc

```diff
@@ -174,6 +176,20 @@ void PromiseReject(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(ret.FromMaybe(false));
 }
 
+void SafeGetenvForJs(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+  Local<String> envtag = args[0]->ToString();
+  String::Utf8Value strenvtag(envtag);
+  std::string text;
+  bool isEnvSafe = false;
```

Comment:
Locals should use snake_case, i.e., `is_env_safe`.

That said, you don't need a separate local, just do `if (!SafeGetenv(...)) return;`